plugins/islandora_context_reaction_restrict_by_ip.inc

```diff
@@ -111,7 +111,7 @@ class IslandoraContextReactionRestrictByIp extends context_reaction {
       if (!strlen($range)) {
         continue;
       }
-      list($low, $high) = explode(':', $range);
+      list($low, $high) = array_pad(explode(':', $range, 2), 2, NULL);
```

Comment:
After looking at the code and using the plugin a bit more, I felt that the current expectation that users provide single IPs using the range notation keeps the lines in the input field more consistent.  So I only fixed the notice, rather than adding additional logic to handle single IP addresses.

I suggest that we merge this fix to the notice, then determine if it's really desirable to mix up single IPs and IP ranges in the input form of the restrict by IP reaction plugin.   Thank you for considering.